core/integration/module_terminal_test.go

```diff
@@ -189,8 +202,8 @@ type Test struct {
 		require.NoError(t, err)
 
 		// cache the returned container so there's less to wait for in the shell invocation below
-		_, err = hostDaggerExec(ctx, t, modDir, "--debug", "call", "ctr", "sync")
-		require.NoError(t, err)
+		// _, err = hostDaggerExec(ctx, t, modDir, "--debug", "call", "ctr", "sync")
+		// require.NoError(t, err)
```

Comment:
is this strictly necessary to comment out, or leftover from testing?

Reply:
Sorry, this was meant to be removed. Since I moved the `Terminal` call into the code itself (rather than CLI invoked), this is causing the terminal to pop open, resulting in test failure.